source/stylist/configuration.py

```diff
@@ -124,7 +124,28 @@ def get_style(self, name: str) -> Sequence[str]:
             if len(rules.strip()) == 0:
                 raise StylistException(f"Style {key} contains no rules")
             else:
-                return rules.split(',')
+                rule_list: List[str] = []
+                state = 'rule'
+                buffer = ''
+                for character in rules:
```

Comment:
Iterating over individual characters feels like a lot of work and quite low level for this, could an approach using something like `rules.split(',')` get you there faster?

Reply:
Each comma separated rule has a comma separated argument list. Thus split on comma doesn't work. It might be possible to do something with regex but while this method is longwinded it is simple. A regex could be fiendishly complicated to understand.